### PR TITLE
Will complete challenge 7

### PR DESCRIPTION
**Pull request by beng355826**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -137,9 +137,93 @@ describe('GET /api/articles/:article_id', () => {
...
+        .post('/api/articles/9000/comments')
+        .send(comment)
+        .expect(404)
+        .then(({body}) => {
+
+        expect(body).toMatchObject({
+            msg:'404 - not found'
+        })
+
+        })
```
Comment (NRMSMITH) [inline __tests__/app.test.js:225]: There's a few other instances I'd like you to think of: 
201 - ignores unnecessary properties on the comment (i.e. it has a username, a body and a colour property)
404 - username does not exist

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
A couple of tests to add please 🙏

File: __tests__/app.test.js (hunk 1 above)
Comment (beng355826) [reply __tests__/app.test.js:252]: Added :)

File: db/app.js [hunk 2]
```diff
@@ -2,40 +2,44 @@ const express = require("express")
...
     })
+
+    next(err)
+})
+
+
+
+app.use((err, req, res, next) => {
+    res.status(500).send({msg: "500 server error"})
 })
```
[the remark this answers is not in the document]
Comment (beng355826) [reply db/app.js]: Done :)

File: __tests__/app.test.js [hunk 3]
```diff
@@ -137,9 +137,141 @@ describe('GET /api/articles/:article_id', () => {
...
+        expect(body.comment).toHaveProperty('created_at', expect.any(String))
+
+        }) 
+    })
+    
 
 
 
 
+    test("sends a 400 when an invalid type request is sent" , () => {
```
Comment (RichardNeat) [inline __tests__/app.test.js:193]: I think we can be clearer with the description here, it took me a minute to work out what this test is doing. Could we say this is for an invalid id

**Review by RichardNeat — COMMENTED** [review]
Really nice Ben! Just one suggested change from me below. All good to merge :)

**Pull request merged**